### DISCUSSION TITLE
Removing myself as a module maintainer

### DISCUSSION THREAD

**Discussion by glitchcrab** [ansible-collections/news-for-maintainers#37] [Discussions on announcements]
I am listed as the maintainer/author of all `memset.*` modules, however I no longer work there (and haven't for a few years). Additionally, Memset was sold to another company who are slowly migrating everything over to their infrastructure so I don't know how long the modules will continue to be of any use for.

As such, would it make sense to delete them entirely? I have no real insight into what's happening at the company anymore so I can't provide much more detail than this.

**Comment by felixfontein** [comment]
Let's continue this in https://github.com/ansible-collections/community.general/pull/6116.

**Comment by mariolenz** [comment]
I think the best thing would be to create a PR in the collection in question that removes yourself as a maintainer.

> As such, would it make sense to delete them entirely? I have no real insight into what's happening at the company anymore so I can't provide much more detail than this.

This is a different question. But, since you stopped working on those modules, I think it's our problem and not yours. But thanks for letting ud know ;-)
